util/throttle.cc

```diff
@@ -30,6 +30,7 @@
 #include "util/cache2.h"
 #include "util/db_list.h"
 #include "util/flexcache.h"
+#include "util/hot_threads.h"
 #include "util/throttle.h"
 
 namespace leveldb {
```

Comment:
comments added

> On Aug 9, 2015, at 10:13 PM, Paul A. Place <notifications@github.com> wrote:
> 
> In util/throttle.cc <https://github.com/basho/leveldb/pull/162#discussion_r36597960>:
> 
> > @@ -30,6 +30,7 @@
> >  #include "util/cache2.h"
> >  #include "util/db_list.h"
> >  #include "util/flexcache.h"
> > +#include "util/hot_threads.h"
> >  #include "util/throttle.h"
> >  
> >  namespace leveldb {
> For lines 41-54, it would be helpful to add some comments about the manifest constants (e.g., why is THROTTLE_SCALING set to 17?), along with a description of how the gThrottledData array is used (e.g., that the first two slots in the array hold live data while the remaining slots hold rolling data from the past hour)
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/basho/leveldb/pull/162/files#r36597960>.
>